roles/StackStorm.nodejs/defaults/main.yml

```
@@ -1,3 +1,3 @@
 ---
 # nodejs version to install
-nodejs_major_version: "10"
+nodejs_major_version: "{% if st2chatops_version == 'latest' or st2chatops_version is version('3.5.0', '>=') %}14{% else %}10{% endif %}"
```

In other words,
```suggestion
nodejs_major_version: "14"
```
is all we need

---

Done. And I updated the README. Hope it's fine now.